src/App.js

```diff
@@ -50,6 +51,7 @@ export default function App() {
           />
           <Route path="/testing" element={<TestingArea />} />
           <Route path="/login" element={<LogIn />} />
+          <Route path="/splashScreen" element={<SplashScreen />}/>
```

Comment:
probably doesn't make sense to have a separate route for the splash screen? I imagine this component will be in the login page eventually